Modules/Session/classes/class.ilSessionMailTemplateParticipantContext.php

```diff
@@ -98,20 +98,25 @@ public function resolveSpecificPlaceholder(
         ilObjUser $recipient = null,
         bool $html_markup = false
     ) : string {
-        /**
-         * @var $ilObjDataCache ilObjectDataCache
-         */
         global $DIC;
 
         $ilObjDataCache = $DIC['ilObjDataCache'];
-
-        if ('crs_title' == $placeholder_id) {
-            return $ilObjDataCache->lookupTitle($ilObjDataCache->lookupObjId($context_parameters['ref_id']));
-        } else {
-            if ('crs_link' == $placeholder_id) {
-                require_once './Services/Link/classes/class.ilLink.php';
-                return ilLink::_getLink($context_parameters['ref_id'], 'crs');
-            }
+        $obj_id = $ilObjDataCache->lookupObjId($context_parameters['ref_id']);
+        $sess_data = ilObjSession::lookupSession($obj_id);
+        $sess_app = ilSessionAppointment::_lookupAppointment($obj_id);
+
+        switch ($placeholder_id) {
+            case 'sess_title':
+                return $ilObjDataCache->lookupTitle($obj_id);
+            case 'sess_appointment':
+                return ilDatePresentation::formatPeriod(
+                    new ilDateTime($sess_app['start'], IL_CAL_UNIX),
```

Comment:
This always presents dates with time information.
Please use ilDate for fullday events and ilDateTime for non fullday  events.

Reply:
Good call! It's now working for both cases :)